src/main/kotlin/com/exactpro/th2/replay/script/generator/ScriptGenerator.kt

```diff
@@ -35,27 +35,29 @@ class ScriptGenerator(
     fun onMessage(message: Message) {
         logger.info { "Processing message: ${message.toJson()}" }
 
-        val transformedMessage = messageTransformer.transform(message)
+        with(messageTransformer.transform(message)) {
```

Comment:
Using `with` won't prevent accidental passing of original message somewhere where it shouldn't be. A better way (but still not ideal) would be to use `let`/`also` with lambda argument name shadowing original message